tests/basic/autopilot_test.go

```diff
@@ -1808,6 +1809,138 @@ func scheduleAppsWithAutopilot(testName string, testScaleFactor int, apRules []a
 	return contexts
 }
 
+//This testsuite is designed to execute fundamental scenarios with Autopilot rules.
+//It involves scheduling applications and waiting until the
+//Autopilot ActiveActionsPendingToActiveActionsInProgress event is triggered.
+//After this event, two storage nodes, one with KVDB, are intentionally crashed,
+//and the test checks whether these nodes successfully recover.
+
+var _ = Describe("{AutoPoolExpandCrashTest}", func() {
+	BeforeEach(func() {
+		StartTorpedoTest(fmt.Sprintf("{%sAutoPoolExpandCrashTest}", testSuiteName), "Crash one kvdb node and one storage node when multiple pools are expanded using autopilot", nil, 0)
+	})
+
+	It("has to crash one kvdb node and one storage node when multiple pools are expanded with add-disk , validate and teardown apps", func() {
+		var contexts []*scheduler.Context
+		testName := strings.ToLower(fmt.Sprintf("%sAutoPoolExpandCrashTest", testSuiteName))
+		poolLabel := map[string]string{"node-type": "fastpath"}
+		crashedNodes := make([]node.Node, 0)
+		storageNode := node.GetStorageDriverNodes()[2]
```

Comment:
use GetStoragenodes if you want storage nodes. GetStorageDriverNodes will return all nodes where PX is running

Reply:
Done